### ISSUE TITLE
Add support for multiple redcap_data tables

### ISSUE THREAD

**Issue by tertek** [lsgs/redcap-drw-tweaks#2]
Use:
```php
$data_table = method_exists('\REDCap', 'getDataTable') ? \REDCap::getDataTable($project_id) : "redcap_data"; // please check if $project_id exists
$sql = "select record, value from $data_table where project_id = " . PROJECT_ID . " and field_name = '__GROUPID__'";
```

for:

https://github.com/lsgs/redcap-drw-tweaks/blob/48a40450aab65a059e1e5d393b22ef023766e8f7/DataQualityTweaked.php#L252

**Comment by lsgs** [comment]
So sorry I missed this @tertek! I thought I'd been through all my EMs and fixed this up a while ago. It's fixed now with release 1.1.0.

**Comment by tertek** [comment]
Thanks for the fix @lsgs !
